packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -282,7 +300,7 @@ const PaymentMethodTitle: FunctionComponent<
                 )}
 
                 {titleText && (
-                    <div className="paymentProviderHeader-name" data-test="payment-method-name">
+                    <div aria-level={6} className="paymentProviderHeader-name" data-test="payment-method-name" role="heading">
```

Comment:
I didn't find any other `aria-level` reference in the codebase.
Can you please explain why `6` is approriate here?

Reply:
Just wanted to stick to a level that's not in top so a screen reader thinks this is a main header.